src/conversations/Conversations.ts

```diff
@@ -429,6 +439,67 @@ export default class Conversations {
     return seenPeers
   }
 
+  async newGroupConversation(
```

Comment:
Are we sure we need this in `xmtp-js`. I was kinda imagining this would all live in `xmtp-group-chat`. Are we relying on some private methods or something?

Reply:
I've been thinking group chat should live here since it's such a core messaging concept and it should be as easy as possible to get a group chat. xmtp-group-chat has just been my playground for testing concepts out. If we feel strongly it should live elsewhere I can go that path though.

Reply:
Thinking about it a bit more, I think putting group chat features into a separate package increases the risk of fragmentation of group chat. Adding the invite capabilities here means that someone might just write their own implementation of group chat if they don't know about the other package. By keeping group chat here we're giving everybody the same set of tools by default.

Reply:
It's your call. Im on board for it living in the SDK. So long as we maintain the bar on documentation and stability for a core protocol feature. The main advantage to having it in its own package is our ability to say "jk! We are changing the protocol and all your old group chats are going to become unreadable". 

Maybe we can use the beta branch to at least buy us a little time to iterate.

Reply:
I dig using the beta branch to iterate in case we end up needing pull the "jk" ripcord, let's go with that.